portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages;
```

Comment:
change this to a final variable, and initialize it in place.

Reply:
Done!

Reply:
did you forget to add a commit? I don't see any changes.